src/frontends/onnx/frontend/src/core/node.cpp

```diff
@@ -203,6 +209,35 @@ const std::string& Node::Impl::description() const {
     return m_description;
 }
 
+template <typename T>
+std::shared_ptr<ov::op::v0::Constant> Node::Impl::get_attribute_as_constant(const std::string& name) const {
+    const auto value = get_attribute_value<T>(name);
+    const element::Type type = element::from<T>();
+    return ov::op::v0::Constant::create(type, Shape{}, std::vector<T>{value});
```

Comment:
you can consider using this constructor https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/op/constant.hpp#L69

Reply:
Good point!  😊  Done